src/Version.php

```diff
@@ -32,11 +32,56 @@ public static function isStable()
     }
 
     /**
-     * Compares a version to Bolt's version.
+     * Compares a semantic version against Bolt's version, given a specified
+     * comparison operator.
      *
-     * Note: Be sure to include the `.z` number in the version given, as omitting it can give inconsistent results.
-     * For example, if the current version is `3.3.0`, `compare('3.3', '>=')` returns false. In reality you want
-     * the check to return true for this case, and `compare('3.3.0', '>=')` _does_ return true.
+     * @see http://semver.org/ For an exmplaination on semantic versioning.
```

Comment:
> IntelliJ/PHPStorm doesn't parse this inline

Then it needs to [update](https://www.phpdoc.org/docs/latest/references/phpdoc/tags/see.html)